Solutions/Marain.Tenancy.Host.Functions/Marain.Tenancy.Host.Functions.csproj

```diff
@@ -10,12 +11,12 @@
   </PropertyGroup>
   
   <ItemGroup>
-    <PackageReference Include="Corvus.Tenancy.Storage.Azure.Blob" Version="0.20.0" />
-    <PackageReference Include="Endjin.RecommendedPractices" Version="0.1.0">
+    <PackageReference Include="Corvus.Tenancy.Storage.Azure.Blob" Version="0.24.0-preview.36" />
+    <PackageReference Include="Endjin.RecommendedPractices" Version="0.2.1">
```

Comment:
Note, due to a bug in the _other_ recommended practices (the ADO pipeline templates), publishing NuGet packages stopped working for a bit, because it was always attempting to call the release something like `$(MajorVersion).$(MinorVersion)` or something of that kind, rather than actually evaluating those variables to the intended numbers...

I have now fixed this, and the latest version of the recommended practices NuGet package is now 0.3.0.